docs/resources/inferred_service_rule.md

```diff
@@ -0,0 +1,62 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "lightstep_inferred_service_rule Resource - terraform-provider-lightstep"
+subcategory: ""
+description: |-
+
+---
+
+# lightstep_inferred_service_rule (Resource)
+
+~> 🚧 This resource is under development and is not generally available yet. 🚧
```

Comment:
Did this get left in here?

Reply:
yeah intentionally! The Public API isn't ready for use yet, so this resource won't work yet. We can remove this line once we Go Live

I was inspired by a similar line on the user binding resource